### ISSUE TITLE
SVG of `Icon` component not shown in Windows High Contrast Mode in Chromium

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1718]
Description  
---
SVG of `Icon` component not shown in Windows High Contrast Mode in Chromium. 
This is a known bug of Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=1242706

Further information: https://melanie-richards.com/blog/currentcolor-svg-hcm/

To Reproduce
---
1. Go to https://designsystem.porsche.com/issue/svg-high-contrast-test/components/icon/examples
2. Activate High Contrast Mode in Windows
3. Icons not visible

Expected behavior
---
Icons should also be visible in High Contrast Mode

Screenshots
---
<img width="1004" alt="Bildschirmfoto 2022-01-12 um 13 03 18" src="https://user-images.githubusercontent.com/40788023/149136952-e4020954-591f-4a3f-8f0b-507895d05be4.png">

Test Environment
---
Platform/Browser/Version: [Windows only, Chrome and Edge]
